app/models/static_content.rb

```diff
@@ -34,7 +34,7 @@ def self.sorted_and_filtered_for_params(params)
   private
 
     def enforce_nil_for_empty_version
-      self.version = nil if version.empty?
+      self.version = nil if version.try(:empty?)
```

Comment:
why is this addressed in this pull request?

Reply:
i stumbled across this when trying to test this feature with an empty db